### DISCUSSION TITLE
Problem with recording

### DISCUSSION THREAD

**Discussion by djp4nd4** [Zealot111/OCAP#14] [Q&A]
[ocap-main.20200417_172841.log](https://github.com/Zealot111/OCAP/files/4493820/ocap-main.20200417_172841.log)
[ocap-ext.20200417_172841.log](https://github.com/Zealot111/OCAP/files/4493821/ocap-ext.20200417_172841.log)

**OCAP Version:** `x.xx`
**Server OS:** `Windows`

**Description:**

**Steps to reproduce:**

**RPT File**

**Comment by tekig** [comment]
Recording will not start until the minimum number of players is reached. https://github.com/Zealot111/OCAP/blob/master/addons/ocap/functions/fn_startCaptureLoop.sqf#L25

Check [userconfig](https://github.com/Zealot111/OCAP/blob/master/userconfig/ocap/config.hpp#L2)

**Comment by djp4nd4** [comment]
thanks, now seem to work fine.
